<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FNatByPeriod</class>
 <widget class="QWidget" name="FNatByPeriod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Period start from</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbYear1">
     <item>
      <property name="text">
       <string>2017</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2018</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="EQComboBox" name="cbMonth1">
     <item>
      <property name="text">
       <string>January</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>February</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mart</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>April</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>May</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>June</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>July</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>August</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>September</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>October</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>November</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>December</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>End of period</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbYear2">
     <item>
      <property name="text">
       <string>2017</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2018</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="EQComboBox" name="cbMonth2">
     <item>
      <property name="text">
       <string>January</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>February</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mart</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>April</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>May</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>June</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>July</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>August</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>September</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>October</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>November</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>December</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>393</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">eqcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
